src/main/kotlin/personDsl/domain/Person.kt

```diff
@@ -0,0 +1,9 @@
+package personDsl.domain
+
+data class Person(
+    val name: String,
+    val company: String?,
+    val softSkills: List<String>,
+    val hardSkills: List<String>,
```

Comment:
```suggestion
    val skills: List<Skill>,
```

Right now, `softSkills` and `hardSkills` are just `List<String>`.
How about wrapping them in a `Skill` class to better represent their meaning in the domain?

(+) As a next step, you could also try using a `sealed interface` with types like `SoftSkill` and `HardSkill`.
It would be a good chance to think about how this approach compares to using an `enum`. 

Here’s a reference you might find helpful:
- [[Kotlin Docs] Sealed classes and interfaces](https://kotlinlang.org/docs/sealed-classes.html)